velox/exec/Task.cpp

```diff
@@ -857,6 +857,8 @@ void Task::recordBatchEndTime() {
 }
 
 void Task::start(uint32_t maxDrivers, uint32_t concurrentSplitGroups) {
+  queryCtx()->setStarted();
```

Comment:
queryCtx is used to provide the query execution context. Why we provide an action method? I have a change to split init from start: https://github.com/facebookincubator/velox/pull/13075. But it breaks meta internal streaming use case. We can work with Meta team if we really needs to support this for server pushdown mechanism? Thanks!